### PR TITLE
robots.txt and sitemap.xml file added for increasing score

### PR DESCRIPTION
**Pull request by Shubhashree3**
Below issues are solved:
1. No robots.txt or X-Robots-Tag HTTP headers found to manage crawler traffic
2. Sitemap.xml to optimize interaction with bots not found
3. No sitemap.xml URL found in robots.txt

### CODE REVIEW

File: robots.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+robots.txt generated by www.seoptimer.com
```
Comment (tapanbavaliya) [inline robots.txt]: @Shubhashree3 remove this line

File: sitemap.xml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  <priority>0.80</priority>
+</url>
+<url>
+  <loc>http://gandhiassociates.in/contact.html</loc>
+  <lastmod>2018-02-27T06:03:11+00:00</lastmod>
+  <priority>0.80</priority>
+</url>
+
+
+</urlset>
```
Comment (tapanbavaliya) [inline sitemap.xml]: add new line at the end of the file

File: robots.txt (hunk 1 above)
Comment (Shubhashree3) [reply robots.txt]: removed this line in both sites.

File: sitemap.xml (hunk 2 above)
Comment (Shubhashree3) [reply sitemap.xml]: added new line in both sites.

**Pull request merged**